cmd/object-handlers.go

```diff
@@ -48,16 +49,37 @@ func setGetRespHeaders(w http.ResponseWriter, reqParams url.Values) {
 	}
 }
 
+// getSourceIPAddress - get the source ip address of the request.
+func getSourceIPAddress(r *http.Request) string {
+	// Attempt to get ip from standard headers.
+	for _, h := range []string{"X-Forwarded-For", "X-Real-Ip"} {
```

Comment:
I used those headers in case a proxy was in front of the request. I thought if that was the case remote address would refer to the proxy not the source of the request. If none of the headers are used it defaults to remoteAddr. Am I wrong?